docs/user-guide/release-notes/2.6.x.rst

```diff
@@ -2,6 +2,18 @@
 Pulp 2.6 Release Notes
 =========================
 
+Pulp 2.6.1
+==========
+
+Bug Fixes
+---------
+
+This is a minor release which contains a fixes for `multiple issues <https://pulp.plan.io/issues?utf8=%E2%9C%93&set_filter=1&f%5B%5D=cf_4&op%5Bcf_4%5D=%3D&v%5Bcf_4%5D%5B%5D=2.6.1&f%5B%5D=tracker_id&op%5Btracker_id%5D=%3D&v%5Btracker_id%5D%5B%5D=1&f%5B%5D=&c%5B%5D=project&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=cf_5&c%5B%5D=subject&c%5B%5D=author&c%5B%5D=assigned_to&c%5B%5D=cf_3&group_by=>`_.
+
+A couple of the bugs were related to upgrades. Celery processes running as `pulp_workers`,
```

Comment:
Replace the first sentence and some of the second with something more proactive. I'll suggest: "One area of improvement relates to upgrades and uninstalls. Starting with 2.6.1, Celery processes...."